R/ms_synthesize.R

```diff
@@ -105,13 +108,16 @@ ms_synthesize = function(
     cat(ssml)
     stop("Need smaller script! SSML is over 1024 characters")
   }
-  res = POST(synth_url,
-             body = ssml,
-             auth_hdr, fmt_hdr, ctype, auth_hdr,
-             ...)
+  res <- POST(url = synth_url,
+              ctype_hdr,
+              fmt_hdr,
+              auth_hdr,
+              user_agent_hdr,
+              host_hdr,
+              body = ssml)
+
   stop_for_status(res)
   out = content(res)
-  class(token) = "token"
```

Comment:
Why was this removed?  This is so printing show off the actual token.

Reply:
Put it back in. My bad